### DISCUSSION TITLE
Non-existent notification

### DISCUSSION THREAD

**Discussion by Desiders** [community/community#111217] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

A few days ago, I noticed that GitHub notifying about a new notification, but I don't have anything in them.
I have tried various filters (All, Unread, Assigned, Mentioned, etc.), but they are empty.

![Screenshot from 2024-03-07 16-25-54](https://github.com/community/community/assets/47452083/c3e5518c-47b6-4039-906f-cef15eca6de6)

**Comment by Brikaa** [comment]
Yes, there seems to have been a spam gist which had a comment that tagged everybody. The gist was then deleted. Maybe GitHub has a bug where the notification bubble stays if the the thing you were subscribed to was deleted and you had an unread notification in it.
![image](https://github.com/community/community/assets/40431124/064a0204-6c64-4abd-80eb-e06797829190)

**Comment by himeshx** [comment]
Hello, I got this same error after being notified about Starknet and Fluence coins airdrop by email. Looks like is related to this: https://github.com/orgs/community/discussions/6874#discussioncomment-8687949

I think that we all were mentioned on a gist, but I wasn't able to look the content.

I already contact Github support and they told me the same about using curl to clear the notifications manually with the API.

For other people who read this answer, please use API to clear your notification box.
```
curl -X PUT \
    -H "Accept: application/vnd.github.v3+json" \
    -H "Authorization: token <TOKEN>" \
     https://api.github.com/notifications \
    -d '{"last_read_at":"2024-03-08T00:00:00Z"}'
```
ps: Kindly consider marking the provided response as the answer if it has addressed your query satisfactorily.

**Comment by Brikaa** [comment, marked as answer]
There seems to have been a deleted comment here which suggested [using GitHub's API to mark the notification as read](https://docs.github.com/en/rest/activity/notifications?apiVersion=2022-11-28#mark-notifications-as-read). Worked for me.

**Reply by adah1972** [reply]
Thank you. I was driven crazy by the ghost notification. Now it is gone.

**Reply by AlexGuo1998** [reply]
I guess this cleared the "unread" state but it is still "not done".
You can confirm this by noticing the `1-0 of 1` text at the bottom-right corner. The usual "All done!" image also disappeared.

**Reply by CTimmerman** [reply]
You can mark it done by using the API as described [here](https://gist.github.com/eseiler/d3a6f978ceea1853f0c558879cf859c1). You can get an access token [here](https://github.com/settings/tokens) if you don't want to install and authorize GitHub's CLI.

**Comment by Desiders** [comment]
I tried to use the API to get a list of notifications, but it's empty:
![](https://github.com/community/community/assets/47452083/b90684c6-db83-4862-ac96-32f359ed3169)

But I tried to read all the API notifications anyway, and it helped.

Thank you, but I don't know if it's necessary to close this discussion if it's a bug?

**Reply by Gargaj** [reply]
It will return empty, but if you set them all to read it will make the ghost notification disappear.

**Comment by Limess** [comment]
If it helps anyone, I just ran this Github CLI command to mark all notifications as read (change the date to todays date to be sure):

```
gh api --method PUT -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /notifications -F read=true
```

**Reply by Gottox** [reply]
Worked for me, thanks!

**Comment by Its-Just-Nans** [comment]
Thanks for the fix

Note for @github : I think there still is a problem in the notifications panel

![image](https://github.com/community/community/assets/56606507/26d4b37b-48fe-4485-9172-c2234382adc2)

**Comment by joshmedeski** [comment]
<img width="2545" alt="SCR-20240603-kntr" src="https://github.com/community/community/assets/2036594/4d158ca3-9499-472d-a682-28a0788fc05d">

I was able to mark the missing notification as "read" but I'm unable to mark it as "done" to get rid of repository in the sidebar. There is no "thread_id" to be able to use the API.

**Reply by CTimmerman** [reply]
You can mark it done by using the API as described [here](https://gist.github.com/eseiler/d3a6f978ceea1853f0c558879cf859c1). You can get an access token [here](https://github.com/settings/tokens) if you don't want to install and authorize GitHub's CLI.
